lib/sqlalchemy/ext/asyncio/engine.py

```diff
@@ -41,6 +41,29 @@ def create_async_engine(*arg, **kw):
     return AsyncEngine(sync_engine)
 
 
+def async_engine_from_config(configuration, prefix="sqlalchemy.", **kwargs):
+    """Create a new AsyncEngine instance using a configuration dictionary.
+
+    This works analogous to :func:`_sa.engine_from_config`, but the
+    configured dialect must be an asyncio-compatible dialect such as
```

Comment:
**mike bayer** (zzzeek) wrote:

This function is analogous to the :func:`_sa.engine_from_config` function
in SQLAlchemy Core, except that the requested dialect must be an 
asyncio-compatible dialect such as :ref:`dialect-postgresql-asyncpg`.
The argument signature of the function is identical to that 
of :func:`_sa.engine_from_config`.

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/3309

Reply:
**Federico Caselli** (CaselIT) wrote:

Ack

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/3309